tool/lib/webqa-manual-annotations.js

```diff
@@ -254,7 +254,9 @@ const MANUAL_PROPERTY_CANONICAL_OVERRIDE = {
         base: ['author', 'creator'],
         passive_verb: [
             'by', 'made by', 'written by', 'created by', 'authored by', 'uploaded by', 'submitted by'
-        ]
+        ],
+        reverse_verb: ['wrote', 'authored']
```

Comment:
made? did? (here and elsewhere)

Reply:
I added "made" to creator field. We will need domain specific override in the future, otherwise some annotations will introduce ambiguity in some domains.